cluster-up/container.sh

```diff
@@ -3,7 +3,9 @@
 set -e
 
 if [[ $KUBEVIRT_PROVIDER =~ (ocp|okd).* ]]; then
-    CONTAINER=$(docker ps | grep kubevirt | grep $KUBEVIRT_PROVIDER | awk '{print $1}')
+    POSTFIX="cluster"
+    PATTERN="${KUBEVIRT_PROVIDER}-${POSTFIX}"
+    CONTAINER=$(docker ps | grep kubevirt | grep $PATTERN | awk '{print $1}')
```

Comment:
would suggest to add a grep cluster and leave it one liner, but up to you,
one liners are good because they can be adapted on other cases as well for example

Reply:
I'd second that, maybe just even compress into one `grep` (with added `cluster`) along the way.

Reply:
I use $PATTERN in order to separate the "what to search" logic form the "how to search" logic, in a way it will be more easy to test and debug in case the container name would change for any reason.
I left "grep kubevirt"  in case that there is for some unexpected reason another container with the same name but different image registry.
I understand if it may sound redundant or over-kill for this small example.

Reply:
I understand you want to make clear that in case of ocp and okd there's a POSTFIX required. I'm fine with that. However, `PATTERN` I find redundant, because grep always searches for a pattern.

Maybe we can settle on sth like this:
```
POSTFIX="cluster"
CONTAINER=$(docker ps | grep "kubevirtci/${KUBEVIRT_PROVIDER}-${POSTFIX}" | awk '{print $1}')
```

Reply:
We still need the "grep kubevirt" because the "cluster" postfix is on the name column and the "kubevirt" string is on the image name column.

Thanks for understanding, I will change the commit.

Reply:
grep -E '1.*2' this can be used if desired one grep, but no biggie,
imo a var on its own line, which always have fixed value is redundant,
even for debug of futuristic cases.

but up to you
LGTM